UPGRADING.md

```diff
@@ -36,6 +39,13 @@
 
 # Upgrading OpenSearch.Net & OpenSearch.Client
 
+## [Unreleased]
+
+### OpenSearch.Client
+
+#### General
+- Namespaced APIs, exposed in `IOpenSearchClient`, have gained an interface, and the properties on `IOpenSearchClient` and `OpenSearchClient` have been changed to the new interfaces. For example, `IOpenSearchClient.Cluster` was `ClusterNamespace` and now is `IClusterNamespace`.
```

Comment:
Sorry I should have given more info, `2.0.0` is the in-development unreleased major version, so this can go in the existing `1.x.y to 2.0.0 > OpenSearch.Client > General` section. 

Wording wise I would potentially rephrase this as:
```suggestion
#### General
- The namespaced APIs exposed in `IOpenSearchClient` have each gained a corresponding interface and the types of the properties on `IOpenSearchClient` and `OpenSearchClient` have been changed from the concrete implementations to the matching interfaces. For example, `IOpenSearchClient.Cluster` was `ClusterNamespace` and now is `IClusterNamespace`.
```

Reply:
This has been updated.